devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -19,6 +19,8 @@ metadata {
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
+		capability "Configuration"
```

Comment:
Why are we adding this capability instead of using the installed() or updated() callbacks?